❤️

```diff
@@ -52,5 +52,6 @@ Sebastian Wachholz
 Stefan van de Weijer
 Thijs Weitkamp
 Tobias van Geijn
+Thomas Giacoletto
```

Comment:
Nitpicking: de lijst is alfabetisch gesorteerd, dus deze change zou 1 regel omhoog moeten.